app/presenters/activity_presenter.rb

```diff
@@ -11,7 +11,7 @@ def aid_type
 
   def aid_type_with_code
     return if aid_type.blank?
-    "#{aid_type} (#{to_model.aid_type})"
+    "#{to_model.aid_type}: #{aid_type}"
```

Comment:
Note to self: `to_model` appears to be acting as a sort of "super" here to bypass the default behaviour of `SimpleDelegator` from which this presenter is derived.